lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -1058,6 +1061,8 @@ private function getColumnNamesInAlteredTable(TableDiff $diff) : array
      */
     private function getIndexesInAlteredTable(TableDiff $diff) : array
     {
+        assert($diff->fromTable !== null);
```

Comment:
Please see if replacing the property access with a method argument would make the API more explicit and eliminate the need for the assertion.

Reply:
Please see above. Doable at first glance. But is it a good practice?